base/shell/explorer/traywnd.cpp

```diff
@@ -2778,6 +2778,10 @@ class CTrayWindow :
             SetWindowPos(hWndInsertAfter, 0, 0, 0, 0, SWP_NOACTIVATE | SWP_NOMOVE | SWP_NOSIZE | SWP_SHOWWINDOW);
         }
 
+        g_TaskbarSettings.bHideInactiveIcons = newSettings->bHideInactiveIcons;
+        g_TaskbarSettings.bShowSeconds = newSettings->bShowSeconds;
+        g_TaskbarSettings.bGroupButtons = newSettings->bGroupButtons;
```

Comment:
These last two lines don't seem related to tray icon hiding. What's their purpose?

Reply:
just to save this flag too. yes, they not related to tray icons

Reply:
oops, sorry ) rollback this